src/posts/posts.11tyData.js

```diff
@@ -0,0 +1,18 @@
+const ALPHABETICAL_RE = new RegExp("[^a-zA-Z0-9 -]");
```

Comment:
This is basically to check if the category has anything apart from alphabets, digits, space and "-", right? The constant name kind of threw me off a bit. Are there specific categories because of which this regex replace was needed?

Reply:
Hey, @SaptakS!

1. yes, that’s what this does
2. For one example, we have a `How-to` category. The corresponding tag as of now is `howto`